dissect/ntfs/mft.py

```diff
@@ -46,6 +46,9 @@ class Mft:
     def __init__(self, fh: BinaryIO, ntfs: Optional[NTFS] = None):
         self.fh = fh
         self.ntfs = ntfs
+        self.record_size = self.ntfs._record_size if self.ntfs else DEFAULT_RECORD_SIZE
```

Comment:
The way this was previously may seem like some code duplication, but there was a very good reason for it. Sometimes in data recovery tooling we may dynamically exchange the `.ntfs` attribute with a different one, possibly changing things like the record size. The correct way to address that would be with a property setter on the `.ntfs` attribute of the `Mft` and `MftRecord` class. Feel free to change that, but it goes beyond the scope of this PR so it's not strictly necessary.